src/FamilyHubs.ServiceDirectory.Core/ServiceDirectory/Interfaces/IServiceDirectoryClient.cs

```diff
@@ -1,14 +1,17 @@
-﻿using FamilyHubs.ServiceDirectory.Core.ServiceDirectory.Models;
+﻿using System.Net;
+using FamilyHubs.ServiceDirectory.Core.ServiceDirectory.Models;
 using FamilyHubs.ServiceDirectory.Shared.Dto;
+using FamilyHubs.ServiceDirectory.Shared.Enums;
 using FamilyHubs.ServiceDirectory.Shared.Models;
-    
+using Microsoft.AspNetCore.Http;
+
 namespace FamilyHubs.ServiceDirectory.Core.ServiceDirectory.Interfaces;
 
 public interface IServiceDirectoryClient
 {
     // leaky, not clean, but this is our service, as opposed to a generic service that we might want to swap
     Task<PaginatedList<TaxonomyDto>> GetTaxonomies(CancellationToken cancellationToken = default);
-    Task<PaginatedList<ServiceDto>> GetServices(ServicesParams servicesParams, CancellationToken cancellationToken = default);
+    Task<Tuple<PaginatedList<ServiceDto>, HttpResponseMessage?>> GetServices(ServicesParams servicesParams, CancellationToken cancellationToken = default);
```

Comment:
Or even breaking out the parts of the response we actually care about? Time / code

Reply:
@Zac-AND-Digital Hm a DTO seems overkill for this since it'd only be used in one place, but I can convert this to a named tuple instead :) That's what I've done on the referral UI:

```c#
Task<(PaginatedList<ServiceDto> services, HttpResponseMessage? response)> ...
```

@Top-Cat I'm a fan of having the HTTP services return their underlying response, rather than just the deserialized response payload. While we technically don't need the whole response, it also means we have a simpler interface (just returning the whole response vs. returning the multiple bits we need)

Happy to discuss and change if anyone has a strong opinion either way :)

Reply:
Makes sense, fine with me :)